packages/fluxible-router/package.json

```diff
@@ -25,15 +25,15 @@
   ],
   "dependencies": {
     "debug": "^2.0.0",
-    "fluxible-addons-react": "^1.0.0-beta.0",
+    "fluxible-addons-react": "^1.0.0-beta.2",
```

Comment:
I dont think this is needed since `^1.0.0-beta.0` should resolve to anything over beta.0. I tested this on https://semver.npmjs.com/ and it seemed to work:

<img width="1041" alt="Screen Shot 2021-05-13 at 9 22 53 AM" src="https://user-images.githubusercontent.com/193272/118155176-cd0fd800-b3cc-11eb-8215-97010f845d44.png">

Reply:
You are completely right. I'm removing it.